packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -154,15 +154,19 @@ const ForwardedInnerBlocks = forwardRef( ( props, ref ) => {
  * returns. Optionally, you can also pass any other props through this hook, and
  * they will be merged and returned.
  *
- * @param {Object} props   Optional. Props to pass to the element. Must contain
- *                         the ref if one is defined.
- * @param {Object} options Optional. Inner blocks options.
+ * @param {Object} props                Optional. Props to pass to the element. Must contain
+ *                                      the ref if one is defined.
+ * @param {Object} options              Optional. Inner blocks options.
  *
+ * @param {string} options.dropZoneName Optional. The name of the drop zone.
  * @see https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/inner-blocks/README.md
  */
 export function useInnerBlocksProps( props = {}, options = {} ) {
-	const { __unstableDisableLayoutClassNames, __unstableDisableDropZone } =
-		options;
+	const {
+		__unstableDisableLayoutClassNames,
+		__unstableDisableDropZone,
+		__unstableDropZoneName = 'inner-blocks',
```

Comment:
`__unstable` because it's meant to be an internal API for now.

I've defaulted it to `inner-blocks` because I think that's the most common case.